soc/infineon/cat1a/Kconfig.soc

```diff
@@ -0,0 +1,119 @@
+# Copyright (c) 2024 Cypress Semiconductor Corporation (an Infineon company) or
+# an affiliate of Cypress Semiconductor Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# MPNs definitions
+rsource "*/Kconfig.soc"
+
+# Infineon CAT1A devices
+
+# Category definitions
+config SOC_FAMILY_INFINEON_CAT1
+	bool
+
+config SOC_FAMILY_INFINEON_CAT1A
+	bool
+
+# Family definitions
+config SOC_FAMILY_PSOC6
+	bool
+	select ARM
+	select CPU_CORTEX_M4
+	select CPU_HAS_ARM_MPU  
+	select DYNAMIC_INTERRUPTS
+	select CPU_HAS_FPU
```

Comment:
this is incorrect, look at how other boards are. SOC_* symbols are selected in Kconfig.soc, anything else is selected in Kconfig

Reply:
updated